woocommerce/woocommerce-ios
# Undo Notice Not Displayed on iPad

Closes: #12335


## Description


- [Make OrderDetailsNotices functions static](https://github.com/woocommerce/woocommerce-ios/pull/12431/commits/3a51f565418941455977f75349c92eb5e515b231) so we do not lose reference to the view controller (order) that was removed which caused the notice not to be presented

## Testing instructions


- Test on both iPad and iPhone
- Open "Orders"
- Select one of the orders in the list
- Scroll to the bottom of order details
- Tap on "Move to trash"
- Confirmation alert will show, tap "Move to Trash"
- Undo notice should show on the bottom of the screen, check that it shows ok
- Tap "Undo" and check that the undo operation was done correctly

## Screenshots


![Simulator Screen Recording - iPad Pro (11-inch) (4th generation) - 2024-04-15 at 09 53 00](https://github.com/woocommerce/woocommerce-ios/assets/6242034/8902470c-c79a-47f7-a9df-08d26cc991b4)

![Simulator Screenshot - iPad Pro (11-inch) (4th generation) - 2024-04-15 at 09 53 22](https://github.com/woocommerce/woocommerce-ios/assets/6242034/2ab4e286-1e49-46d5-9967-09fceaf6ad81)

---
